src/firecfg/firecfg.config

```
@@ -23,6 +23,7 @@ Maps
 Mathematica
 Natron
 PCSX2
+Postman
 PPSSPPQt
 PPSSPPSDL
```

```suggestion
PPSSPPQt
PPSSPPSDL
Postman
```

This file appears to use C/ASCII collation, so sort using that.

(Sorry for the back-and-forth)

---

> This file appears to use C/ASCII collation, so sort using that.

Interesting. If there's a way to use an easier sorting mechanism we should look into that IMO.